src/components/renderer.cpp

```diff
@@ -300,13 +276,12 @@ void renderer::end() {
     fill_background();
   }
 
-
   // For pseudo-transparency, capture the contents of the rendered bar and
   // composite it against the desktop wallpaper. This way transparent parts of
   // the bar will be filled by the wallpaper creating illusion of transparency.
   if (m_pseudo_transparency) {
     cairo_pattern_t* barcontents{};
-    m_context->pop(&barcontents); // corresponding push is in renderer::begin
+    m_context->pop(&barcontents);  // corresponding push is in renderer::begin
```

Comment:
Why? :thinking:

Reply:
Sorry, my IDE made this modification because of the clang format.
I forgot to remove that from the commit.
However this is the current setting in the clang format.
The current value is 2 (Google default).
Maybe the clang format should be modified as follow:
```
SpacesBeforeTrailingComments: 1
```

Reply:
I can remove that and add this line to the clang format if you want.

Reply:
Yes, please do :)